app/src/main/java/com/example/easycloset/MainActivity.java

```diff
@@ -191,4 +196,13 @@ public void onClick(DialogInterface dialog, int id) {
         alertDialog.show();
     }
 
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, @Nullable Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        if (requestCode == 1 && resultCode == RESULT_OK) {
```

Comment:
please define constant for request code

Reply:
fixed by define a constant variable for requestCode